pkg/ui/workspaces/cluster-ui/src/api/txnInsightsApi.ts

```diff
@@ -151,56 +151,36 @@ export type TransactionContentionEventDetails = Omit<
 >;
 
 // txnContentionDetailsQuery selects information about a specific transaction contention event.
-function txnContentionDetailsQuery(filters: TxnContentionDetailsRequest) {
-  const whereClause = getTxnContentionWhereClause(
-    ` WHERE waiting_txn_id = '${filters.txnExecutionID}'`,
-    filters,
-  );
+export function txnContentionDetailsQuery(whereClause: string) {
```

Comment:
nit: can we convert this `whereClause` param to an object with filters, and build the where clause in this function?